src/main/java/cc/hyperium/config/Settings.java

```diff
@@ -605,6 +605,18 @@
     @SliderSetting(name = "gui.settings.fpslimiteramount", isInt = true, min = 5, max = 60, category = IMPROVEMENTS)
     public static int FPS_LIMITER_AMOUNT = 30;
 
+    @ConfigOpt
+    @ToggleSetting(name = "gui.settings.disablelightning", category = IMPROVEMENTS)
+    public static boolean DISABLE_LIGHTNING = false;
+
+    @ConfigOpt
+    //@ToggleSetting(name = "gui.settings.keeplightinguhc", category = IMPROVEMENTS) unable to figure out how to check the users game they're playing, will add back later
+    public static boolean UHC_LIGHTNING = false;
```

Comment:
Try MinigameListener ->  getCurrentMinigameName()

Reply:
I expect for UHC you're checking for "UHC CHAMPIONS" since it reads from the scoreboard.

Reply:
Or whatever method the discord rp uses

Reply:
k thats resolved